docs/modules/core/pages/collision/terrain_collision.adoc

```diff
@@ -156,7 +156,7 @@ public class HelloTerrainCollision extends SimpleApplication
 
     // You can change the gravity of individual physics objects after they are
     // added to the PhysicsSpace.
-    player.setGravity(new Vector3f(0,-30f,0));
+    player.setGravity(-30f);
```

Comment:
I think gravity should be a positive value when used with `setGravity(float)`. 

@stephengold, am I right?

Reply:
You're right, @Ali-RS. The `PhysicsCharacter.setGravity(float)` argument represents acceleration in the character's "down" direction, so i's usually non-negative. That's true in both Minie and jme3-jbullet.

It would be great to get all the tutorials in the Wiki synched with the "jme3test.helloworld" package in jme3-examples.